dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/PushNotificationsLRP.idl

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+import "oaidl.idl";
+import "ocidl.idl";
+
+#include "../PushNotifications-Constants.h"
+
+[object]
+[uuid(60FC21B2-B396-4D49-94F0-7555869FB93C)]
+[pointer_default(unique)]
+interface IWpnLrpPlatform : IUnknown
+{
+    HRESULT RegisterActivator([in] LPCWSTR processName);
```

Comment:
Have we finalized on the API signatures here? I don't believe some of these are accurate. For example, RegisterForegroundActivator additionally takes in a IWpnForegroundSink argument in it's definition. 

For now I would just add one API whose definition I am certain about, implement it and see how it integrates with the lifetime changes you are making. A good candidate is RegisterFullTrustApplication. I would also add a corresponding UnRegisterFullTrustApplication.

Reply:
@sharath2727 and @pmpurifoy please chime in and make sure that the signatures fit with your implementation plans.

Reply:
To start of with, For registerapplication we would need to pass three parameters: input: remoteId, processname and output: appIdentifier

we use the processname to extract the appIdentifier but we need the remoteId and appldentifier to register.
Thanks.